lib/usbmux/index.js

```diff
@@ -51,6 +53,14 @@ function swap16 (val) {
  */
 async function getDefaultSocket (socketPath = DEFAULT_USBMUXD_SOCKET, timeout = 5000) {
   if (!await fs.exists(socketPath)) {
```

Comment:
I see quite a lot of code duplication here. It makes sense to rewrite this method, so it becomes more universal:

```js
async function getDefaultSocket (opts = {}) {
  const {
    socketPath = DEFAULT_USBMUXD_SOCKET,
    socketPort = DEFAULT_USBMUXD_PORT,
    socketHost = '127.0.0.1',
    timeout = 5000,
  } = opts;

  let socket;
  if (await fs.exists(socketPath)) {
    socket = net.createConnection(socketPath);
  } else if (process.platform === 'linux' && /microsoft/i.test(os.release())) {
    // Connect to usbmuxd when running on WSL1
    socket = net.createConnection(socketPort, socketHost);
  } else {
    throw new Error(`The usbmuxd socket at '${socketPath}' does not exist or is not accessible`);
  }

  return await new B((resolve, reject) => {
    socket.once('error', reject);
    socket.once('connect', () => resolve(socket));
  }).timeout(timeout);
}
```